tests/testthat/test_post_alignment.R

```diff
@@ -177,3 +177,121 @@ test_that("test_setOtherPrecursors", {
+                                        4735.13, 4347.61, 4425.08, NA),
+                                 intensity = c(396.068, 48.3087,
+                                               54.6872, 60.1767, 57.5545, NA, 933.868, 96.0476, 100.972,
+                                               78.625, 15.4901, NA, 412.235, 44.3814, 46.9198, 15.0419,
+                                               64.7877, NA),
+                                 leftWidth = c(4620.22119140625, 4582.669921875,
+                                                4698.740234375, 4347.1201171875, 4879.6767578125, NA, 4637.30078125,
+                                                4613.40380859375, 4722.64599609375, 4838.7158203125, 4473.43994140625,
+                                                NA, 4647.56005859375, 4616.8359375, 4726.078125, 4336.90380859375,
+                                                4401.77001953125, NA),
+                                 rightWidth = c(4691.91015625, 4620.22119140625,
+                                                4743.1220703125, 4388.0849609375, 4906.98779296875, NA, 4708.98876953125,
+                                                4644.1279296875, 4767.02587890625, 4889.923828125, 4487.09521484375,
+                                                NA, 4709.009765625, 4647.56005859375, 4760.216796875, 4353.97412109375,
+                                                4446.14697265625, NA),
+                                 peak_group_rank = c(1L, 2L, 3L, 4L,
+                                                     5L, NA, 1L, 2L, 3L, 4L, 5L, NA, 1L, 2L, 3L, 4L, 5L, NA),
+                                 m_score = c(5.69207721480095e-05, 0.0157610817708339, 0.224875109435837,
+                                             0.281696713296988, 0.30530712857306, NA, 5.69207721480095e-05,
+                                             0.00535845032028004, 0.0219196954403725, 0.318687030402833,
+                                             0.383338141620706, NA, 5.69207721480095e-05, 0.0273833236082787,
+                                             0.140154915469226, 0.213467945793095, 0.327528954268794,
+                                             NA),
+                                 run = c("run0", "run0", "run0", "run0", "run0", "run0",
+                                         "run1", "run1", "run1", "run1", "run1", "run1", "run2", "run2",
+                                         "run2", "run2", "run2", "run2"),
+                                 alignment_rank = c(1L, NA,
+                                                    NA, NA, NA, NA, 1L, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,
+                                                    NA)), row.names = c(NA, -18L), class = c("data.table", "data.frame"
+                                                    ), sorted = "run")
+            # Aligned RT axes
+            tAligned <- structure(c(4313, 4316.4, 4319.8, 4323.2, 4326.6, 4330.1, 4333.5,
```

Comment:
I would move this to utils.R so that it could be used in other test-cases. Otherwise, one has to define it in everyfile which may become tedious given the length of the vector

Reply:
done.